BookPlayer/Utils/Storyboarded.swift

```diff
@@ -9,7 +9,7 @@
 import UIKit
 
 enum StoryboardName: String {
-  case Main, Settings, Player
+  case Main, Settings, Player, Speed
```

Comment:
This change is not necessary

Reply:
strange. git didn't add this file, updated

Reply:
removed